src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/core/AbstractCraftingRequestResolver.java

```diff
@@ -242,20 +245,62 @@ protected boolean createsCraftingCycle(
             return null;
         }
 
-        return createRequestsForRecipe(manager, craftableCrafting.getPrimaryOutput(), count, minCount);
+        return createRequestsForRecipe(manager, craftableCrafting, count, minCount);
     }
 
+    /**
+     * Create the crafting request entries for the overall request
+     * Will produce multiple, if the ingredients don't all fit in the crafters inventory. 
+     * @param manager
```

Comment:
If you add the javadoc, then also make it complete.

Reply:
Done